wicketstuff-pdf-js-parent/wicketstuff-pdfjs/src/main/java/org/wicketstuff/pdfjs/res/wicketstuff-pdf.js

```diff
@@ -39,12 +42,20 @@
                 pageRendering = false,
                 pageNumPending = null,
                 scale = config.initialScale || 0.8,
+                autoScale = "",
                 canvas = document.getElementById(config.canvasId),
-                ctx = canvas.getContext('2d');
+                ctx = canvas.getContext('2d'),
+                abortingPrinting = false;
 
             var MIN_SCALE = 0.25;
             var MAX_SCALE = 10.0;
             var DEFAULT_SCALE_DELTA = 1.1;
+            var MAX_AUTO_SCALE = 1.25;
+
+            var container = $(".pdfPanel");
```

Comment:
This should be scoped to the panel related to the current `canvasId` because there might be more than one PdfPanels on the page.